test/parallel/test-dgram-send-queue-info.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+const cp = require('child_process');
+
+const socket = dgram.createSocket('udp4');
+assert.strictEqual(socket.getSendQueueSize(), 0);
+assert.strictEqual(socket.getSendQueueCount(), 0);
+
+const CODE = `
+  const assert = require('assert');
+  const dgram = require('dgram');
+  const server = dgram.createSocket('udp4');
+  const client = dgram.createSocket('udp4');
+  server.bind(0, () => {
+    client.connect(server.address().port, () => {
+      const data = "hello";
+      client.send(data);
+      client.send(data);
+      assert.strictEqual(client.getSendQueueSize(), data.length * 2);
+      assert.strictEqual(client.getSendQueueCount(), 2);
+      client.close();
+      server.close();
+    });
+  });
+`;
+
+const proc = cp.spawn(process.execPath,
+                      [ '--test-udp-no-try-send',
```

Comment:
Would it make sense to put a `// Flags: --test-udp-no-try-send` line at the top of this file instead and avoid the child process workaround?

Reply:
Like this ?
```
// Flags: --test-udp-no-try-send
'use strict';
const common = require('../common');
const assert = require('assert');
const dgram = require('dgram');

const socket = dgram.createSocket('udp4');
assert.strictEqual(socket.getSendQueueSize(), 0);
assert.strictEqual(socket.getSendQueueCount(), 0);
socket.close();

const server = dgram.createSocket('udp4');
const client = dgram.createSocket('udp4');

server.bind(0, common.mustCall(() => {
  client.connect(server.address().port, common.mustCall(() => {
    const data = "hello";
    client.send(data);
    client.send(data);
    assert.strictEqual(client.getSendQueueSize(), data.length * 2);
    assert.strictEqual(client.getSendQueueCount(), 2);
    client.close();
    server.close();
  }));
}));
```